nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -987,14 +994,14 @@ define(
                 });
 
                 fsm.bus.on('on-error', () => {
-                    setExecMessage('');
+                    updateJobState();
```

Comment:
The app cell wasn't show anything in the title bar if the job ended in an error. This ensures that there's a message to indicate the job is kaput.